lib/signal_tower.ex

```diff
@@ -16,6 +16,10 @@ defmodule SignalTower do
   end
 
   defp start_cowboy() do
+    ip = case System.get_env("SIGNALTOWER_LOCALHOST") do
```

Comment:
in Elixir, "if" is an expression (instead of a statement) as well, so this could (if it's not too long) be simplyfied to a oneliner:
ip = if System..., do: {127,0,0,1}, else {0,0,0,0}

Reply:
mix format will not allow a one liner with if

Reply:
Is there a best practice whether to use case or if/else in elixir?

Reply:
if it's about a boolean, I think I would choose if because it's simpler to think about?!